pwndbg/commands/ai.py

```diff
@@ -364,6 +415,7 @@ def ai(question, model, temperature, max_tokens, verbose, list_models=False, com
     # print the arguments
     global last_question, last_answer, last_pc, last_command, verbosity
     ai_openai_api_key = get_openai_api_key()
+    ai_anthropic_api_key = get_anthropic_api_key()
```

Comment:
It seems that `ai_anthropic_api_key` variable is unused?

Reply:
ah, ok, fixed. i should be checking to see if *either* API key variable is set in the if statement that follows this.